test/test-search.ts

```diff
@@ -2,16 +2,16 @@ import test from 'ava';
 import * as search from '../src/search';
 
 test(
-    'searchValue should have http module in standard require' +
-        'http case',
+    `searchValue should have http module in standard require
+        http case`,
```

Comment:
This will make the test string appear strangely because it has a newline. Does this fit in one line?

Reply:
For other ones, consider keeping the `+`, or use
```
['myLongString',
'myLongString2'
].join('')

Reply:
done